CHANGELOG.md

```diff
@@ -2,6 +2,90 @@
 
 <!-- next version -->
 
+## 0.111.0
+
+### 💡 Enhancements 💡
+
+- `auto-instrumentation`: set OTEL_LOGS_EXPORTER env var to otlp in python instrumentation (#3330)
+
+- `collector`: Expose the Collector telemetry endpoint by default. (#3361)
+  The collector v0.111.0 changes the default binding of the telemetry metrics endpoint from `0.0.0.0` to `localhost`.
+  To avoid any disruption we fallback to "0.0.0.0:{PORT}" as default address.
+  Details can be found here: [opentelemetry-collector#11251](https://github.com/open-telemetry/opentelemetry-collector/pull/11251)
+
+
+- `auto-instrumentation`: Add support for specifying exporter TLS certificates in auto-instrumentation. (#3338)
+  Now Instrumentation CR supports specifying TLS certificates for exporter:
+  ```yaml
+  spec:
+    exporter:
+      endpoint: https://otel-collector:4317
+      tls:
+        secretName: otel-tls-certs
+        configMapName: otel-ca-bundle
+        # otel-ca-bundle
+        ca_file: ca.crt
+        # present in otel-tls-certs
+        cert_file: tls.crt
+        # present in otel-tls-certs
+        key_file: tls.key
+  ```
+
+  * Propagating secrets across namespaces can be done with https://github.com/EmberStack/kubernetes-reflector or https://github.com/zakkg3/ClusterSecret
+  * Restarting workloads on certificate renewal can be done with https://github.com/stakater/Reloader or https://github.com/wave-k8s/wave
+
+- `collector`: Add native sidecar injection behind a feature gate which is disabled by default. (#2376)
+  Native sidecars are supported since Kubernetes version `1.28` and are availabe by default since `1.29`.
```

Comment:
```suggestion
- `collector`: Add native sidecar injection behind a feature gate which is disabled by default. (#2376)
  
  Native sidecars are supported since Kubernetes version `1.28` and are availabe by default since `1.29`.
```